src/UCTSearch.cpp

```diff
@@ -196,18 +196,16 @@ Move UCTSearch::get_best_move() {
         return bestmove;
     }
 
-    // should we consider resigning?
-    /*
+    // should we consider resigning?    
        float bestscore = m_root->get_first_child()->get_eval(color);
        int visits = m_root->get_visits();
     // bad score and visited enough
     if (bestscore < ((float)cfg_resignpct / 100.0f)
         && visits > 500
```

Comment:
That 500 visits was there. I didn't put that in. It makes sense to have a threshold to keep it from resigning when it can't get enough playouts to determine if its a good move though. Default resignpct should probably be 0, then it won't resign, unless overridden...

Reply:
I definitely think there needs to be a minimum visit threshold, I just wonder if it should be calculated rather than a constant.  Its much easier to get 500 visits to a 'best' move which is actually almost tied three ways, if the actual number of visits is 1600, rather than 800.  This is relevant if the other moves were discovered late and have good win rates but haven't quite caught up to the leader by the 1600 visits. Temp is actually more likely to choose a move other than this one, since its less than half the visits.
Maybe > half the visits on the one option, and also > 500 to have confidence that the eval is reasonably calculated. (Or maybe all options with > 500 visits must agree to resign? Or the weighted majority of options with > 500 visits?  I don't know what is best.)

Reply:
This condition was removed upstream. It dates back to the time when the score was estimated from Monte Carlo playouts, and doesn't make a lot of sense with a strong neural network evaluator.